Add N&N for Expand All/Collapse All in PDE preference pages (#552)
eclipse-platform/www.eclipse.org-eclipse: news/4.40/pde.md

## API Tools
-->

> 

Reviewer: It seems odd to put it in this category because I'd expect some new compiler option here.  

Probably views and dialogs is more appropriate.

Author: Done.

## PDE Compiler 

> ### New Expand and Collapse toolbar options for PDE Preference Pages

Reviewer: Title case please.  And something shorter.

```suggestion
### Expand-all and Collapse-all Support in Preference Pages
```

Author: Done.

<details>
<summary>Contributors</summary>

- [Elsa Zacharia](https://github.com/elsazac)
</details>

> New toolbar options to expand and collapse all preference sections have been added to the `Compilers` and `API Errors/Warnings` pages under Plug-in Development.

Reviewer: It seems strange to describe them as options. 

```suggestion
A new toolbar to support expanding and collapsing all preference sections has been added to the `Compilers` and `API Errors/Warnings` pages under `Plug-in Development`.
```

Author: Done.
